### ISSUE TITLE
Buttons must have discernible text (.btn-round)

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#91]
Issue
Buttons must have discernible text (button-name)
Target application
API Management
Element path
iframe;article:nth-child(10) > .row > .col-md-12.col.samples > .style-card-empty.style-card > .btn-round
Snippet
<button class="btn-round" data-bind="click: $component.openInEditor.bind($data, item.name)"></button>
How to fix
Fix any of the following:
- Element does not have inner text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"
- Element has no title attribute or the title attribute is empty
Environment

**Comment by azaslonov** [comment]
Fixed.
